roles/terrible/tasks/assert_vars.yml

```diff
@@ -264,7 +247,7 @@
   assert:
     quiet: yes
     that:
-      - "{{ item }}.mount_point | regex_search('^/\\w+')"
+      - '{{ item }}.mount_point == "none" or {{ item }}.mount_point | regex_search("^/\\w+")'
```

Comment:
We should also introduce the assert to control that at least one swap disk has been added. What do you think?

Reply:
No, it's not mandatory to have swap